RFCs/0011-partiql-insert-update-spec.md

```diff
@@ -0,0 +1,907 @@
+- Start Date: 2022-06-17
+- PartiQL Issue [partiql/partiql-docs/#4](https://github.com/partiql/partiql-docs/issues/4)
+- RFC PR: [partiql/partiql-docs/#11](https://github.com/partiql/partiql-docs/pull/11)
+
+# Summary
+
+[summary]: #summary
+
+This RFC specifies PartiQL's syntax and semantics for insert-or-update—also known as UPSERT—as part of PartiQL 
```

Comment:
Very closely related to INSERT/UPSERT is REPLACE and this PR also handles REPLACE so it would be good to call it out here.

Reply:
Sure, added.